pkg/operator/ceph/pool/validate.go

```diff
@@ -81,13 +84,24 @@ func ValidatePoolSpec(context *clusterd.Context, clusterInfo *client.ClusterInfo
 		if !found {
 			return errors.Errorf("unrecognized crush root %s", p.CrushRoot)
 		}
+	} else {
+		// set the crush root to the default if not already specified
+		p.CrushRoot = cephv1.DefaultCRUSHRoot
 	}
 
 	// validate pool replica size
 	if p.Replicated.Size == 1 && p.Replicated.RequireSafeReplicaSize {
 		return errors.Errorf("error pool size is %d and requireSafeReplicaSize is %t, must be false", p.Replicated.Size, p.Replicated.RequireSafeReplicaSize)
 	}
 
+	if p.Replicated.Size <= p.ReplicasPerFailureDomain && p.ReplicasPerFailureDomain != 0 {
+		return errors.Errorf("error pool size is %d and replicasPerFailureDomain is %d, size must be greater, by a power of 2", p.Replicated.Size, p.ReplicasPerFailureDomain)
+	}
+
+	if p.ReplicasPerFailureDomain%2 != 0 && p.ReplicasPerFailureDomain != 0 {
+		return errors.Errorf("error replicasPerFailureDomain is %d must be a power of 2", p.ReplicasPerFailureDomain)
```

Comment:
Why does it have to be a multiple of two? Does ceph require that? 
```suggestion
		return errors.Errorf("error replicasPerFailureDomain is %d must be a multiple of 2", p.ReplicasPerFailureDomain)
```

Reply:
Not really but I think we want to follow the path of simplicity. So replica 4 and replicasPerFailureDomain 2 for instance.